README.md

```diff
@@ -255,34 +159,22 @@ If you like this project...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+ I would like to thank the Microverse team for their great support!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
 
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
+## ❓ FAQ <a name="faq"></a>
 
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+Will update soon...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly fix the broken license MIT link and add your own license.

Reply:
This issue has been resolved. Thanks